.circleci/config.yml

```diff
@@ -73,6 +73,10 @@ jobs:
 workflows:
   version: 2
```

Comment:
Does this have to be `2`? What happens if it's `2.1`, or left off?

Reply:
AFAIK, `2` is the latest version for workflows, and is the default (note that this is different from the global circleci version, which is at the top)